openapi/kas-fleet-manager.yaml

```diff
@@ -1021,6 +1021,10 @@ components:
               type: string
             max_connection_attempts_per_sec:
               type: integer
+            billing_cloud_account_id:
+              type: string
+            marketplace:
+              type: string
```

Comment:
We might need to bump the version https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/1ae59b0752dcd500ab20ae567061ea601c8ede2e/openapi/kas-fleet-manager.yaml#L5

Reply:
:+1: